tests/entrypoints/openai/test_chat.py

```diff
@@ -486,6 +486,47 @@ async def test_chat_completion_stream_options(client: openai.AsyncOpenAI,
     assert last_completion_tokens == 10
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "model_name",
+    ["HuggingFaceH4/zephyr-7b-beta", "zephyr-lora"],
+)
+async def test_chat_with_enable_force_include_usage(client: openai.AsyncOpenAI,
```

Comment:
I could not run this test locally on my machine 😅 

```
torch.OutOfMemoryError: CUDA out of memory. Tried to allocate 224.00 MiB. GPU 0 has a total capacity of 11.72 GiB of which 182.06 MiB is free. Including non-PyTorch memory, this process has 11.47 GiB memory in use. Of the allocated memory 11.30 GiB is allocated by PyTorch, and 13.48 MiB is reserved by PyTorch but unallocated. If reserved but unallocated memory is large try setting PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True to avoid fragmentation.  See documentation for Memory Management  (https://pytorch.org/docs/stable/notes/cuda.html#environment-variables)
```